### PR TITLE
Read home permissions and root aliases

### PR DESCRIPTION
**Pull request by dgdavid**
Adapt Linux readers to read home permissions and root aliases too.

* https://trello.com/c/UIMO6uJg/2655-1-missing-atributes-on-linuxreader

### CODE REVIEW

File: src/lib/y2users/linux/base_reader.rb [hunk 1]
```diff
@@ -42,9 +42,6 @@ def read
         Config.new.tap do |config|
           read_elements(config)
           read_passwords(config)
-          read_authorized_keys(config)
```
Comment (joseivanlopez) [inline src/lib/y2users/linux/base_reader.rb:45]: Please, remove the unneeded requires (i.e, login_config, useradd_config_reader and ssh_authorized_keyring).

File: src/lib/y2users/linux/reader.rb [hunk 2]
```diff
@@ -20,11 +20,93 @@
...
+      #
+      # @return [Array<Y2Users::User>]
+      def read_home_permissions(config)
+        config.users.reject(&:system?).each do |user|
+          next unless user.home && Dir.exist?(user.home.path)
+
+          user.home.permissions = Yast::Execute.on_target!(
+            STAT, "--printf", "%a", user.home.path,
+            stdout: :capture
+          ).prepend("0")
```
Comment (joseivanlopez) [inline src/lib/y2users/linux/reader.rb:73]: Please, add a comment to explain the result of start %a and why it is needed to prepend a 0.

File: src/lib/y2users/linux/base_reader.rb (hunk 1 above)
Comment (dgdavid) [reply src/lib/y2users/linux/base_reader.rb:45]: Arrrrgghhh! :man_facepalming: :man_facepalming: :man_facepalming:

File: src/lib/y2users/linux/reader.rb (hunk 2 above)
Comment (dgdavid) [reply src/lib/y2users/linux/reader.rb]: While trying to write down _why it is needed_ I realized that using the [`#` printf flag character](https://www.man7.org/linux/man-pages/man3/printf.3.html) it's better. Honestly, [`stat`documentation](https://github.com/coreutils/coreutils/blob/a0468760ffa9f7843bbd0d6cf60685f63d33b600/src/stat.c#L1783) was not so clear to me the first time I read it.

Please, check the added comment and suggest changes if you find it confusing.

Thanks!

File: src/lib/y2users/linux/reader.rb [hunk 3]
```diff
@@ -19,12 +19,101 @@
...
+      private_constant :STAT
+
+      # Reads home permissions
+      #
+      # @return [Array<Y2Users::User>]
+      def read_home_permissions(config)
+        config.users.reject(&:system?).each do |user|
+          next unless user.home && Dir.exist?(user.home.path)
+
+          # NOTE: '#' printf flag is needed to make sure permissions bits
```
Comment (joseivanlopez) [inline src/lib/y2users/linux/reader.rb:74]: I would simply this:

~~~
# "stat --printf %#a" returns the permissions in octal format (e.g., 0755), as expected by {Home#permissions}. 
~~~

**Review by joseivanlopez — APPROVED** [review]
LGTM

**Pull request merged**